third-party/linshare/src/test/java/org/apache/james/linshare/client/LinshareAPITest.java

```diff
@@ -173,20 +177,20 @@ void shareShouldTriggerAnEmail() throws Exception {
             .untilAsserted(
                 () -> request
                     .get("/api/email")
-                .then()
+                    .then()
```

Comment:
`then()` shouldn't stay indented the way it was, it helps increasing readability of chained operations